coders/heic.c

```diff
@@ -85,6 +86,8 @@ static MagickBooleanType
   WriteHEICImage(const ImageInfo *,Image *,ExceptionInfo *);
 #endif
 
+static Image *CompensateOrientation(Image *image, ExceptionInfo *exception);
```

Comment:
Could you move this method above the `ReadHEICImage` method, near `IsHeifSuccess`? We can then remove this line.

Reply:
Of course, will do 👍